agent_issuance/src/server_config/aggregate.rs

```diff
@@ -48,11 +55,49 @@ impl Aggregate for ServerConfig {
                 credential_issuer_metadata,
             }]),
 
-            CreateCredentialsSupported {
-                credential_configurations_supported,
-            } => Ok(vec![CredentialsSupportedCreated {
-                credential_configurations_supported,
-            }]),
+            AddCredentialConfiguration {
+                credential_configuration_id,
+                credential_format_with_parameters,
+                display,
+            } => {
+                let cryptographic_binding_methods_supported =
+                    config!("subject_syntax_types_supported", Vec<String>).unwrap_or_default();
+
+                let credential_signing_alg_values_supported =
+                    config!("signing_algorithms_supported", Vec<String>).unwrap_or_default();
```

Comment:
Als dit keuzes zijn die ook op meerdere plekken gebruikt kunnen worden, wellicht enum gebruiken zodat we typo's voorkomen. Tenzij je zeker weer dat dit nergens anders voorkomt

Reply:
I want to change the whole way the `config` macro works eventually. Most likely I just want to have a:
`pub static TEST_METADATA: std::sync::RwLock<Option<Config>> = std::sync::Mutex::new(None);` 

static through which `config!` can fetch variables, Something like `let test = config!(signing_algorithms_supported);`

where:
```rust
struct Config {
    signing_algorithms_supported: Vec<String>,
    ...
}
```

When I find the time :sweat_smile: